app/js/main.js

```diff
@@ -800,59 +806,35 @@ function saveFrame(id) {
 }
 
 /**
- * Confirm the action to be performed.
- *
- * @param {Function} callback The function to run on "OK" being pressed.
- * @param {*} args Arguments of function to run.
- * @param {String} msg Message to display in confirm dialogue.
+ * Creates a dialogue box
+ * @param {Object} swalArgs The SweetAlert arguments to use.
+ * @returns {Promise} Returns a Promise with the outcome of the dialogue.
+ *                    Resolves true if confirm was selected and null if the alert was dismissed.
  */
-function confirmSet(callback, args, msg) {
+function confirmSet(swalArgs) {
   "use strict";
-  confirmText.innerHTML = msg;
-  confirmContainer.classList.remove("hidden");
-  btnConfirmOK.focus();
+  // Set default SweetAlert argument values
+  swalArgs.title = ("title" in swalArgs) ? swalArgs.title : "Confirm";
```

Comment:
`("title" in swawlArgs)` is more of a Python convention. Since you're only accessing a top-level prop, change it and the other lines to this format:
```suggestion
  swalArgs.title = swalArgs.title ? swalArgs.title : "Confirm";
```